Applications/NodeConf-TLP.md

```diff
@@ -0,0 +1,59 @@
+## Introduction
+
+## History & Metrics
+
+NodeConf was started in 2011 by Mikeal Rogers (@mikeal) and Chris Williams (@voodootikigod).
+
+Today there are several NodeConf's all run by differnet organizers who loosely support each other in the `nodeconf` GitHub org.
+
+## Scope
+
+All current and future NodeConf's:
+
+* NodeConf US (currently in hiatus)
+* NodeConf Adventure
+* NodeConf EU
+* NodeConf London
+* NodeConf Barcelona
+* NodeConf Christchurch
+* NodeConf Santa Cruz
+* NodeConf Buenos Aires
+* NodeConf Austin
+* NodeConf Oakland
+* NodeConf Oslo
+
+## Governance
+
+Individual NodeConf's are considered Working Groups under the Foundation's Governance model.
+
+The NodeConf Community is independently governed by existing conference organizers. The TC (Technical Committee) will use the 
+consensus seeking process that has become the standard in Node.js Core for some time.
+
+A single seat on the TC is appointed by each Conference WG.
+
+## Contribution Process
+
+Anyone interested in running a NodeConf can create an issue on the `organizers` repo (currently called the `collective` repo).
+If no objections are raised by the TC a new team and repo will be created that is owned by the requesting organizers.
+
+
```

Comment:
nah, I'm on a Chromebook while my Macbook gets fixed so I'm using Cloud9 and some things look right in terms of spacing when they shouldn't. This will get fixed.